openmsupply/open-msupply
# 1626 remote authorisation   restrict response requisition edibility

Fixes #1626 

# 👩🏻‍💻 What does this PR do? 
Backend restrictions for response requisitions if requisition needs approval status and disable edibility in UI.

# 🧪 How has/should this change been tested? 
Run tests in backend.
- Create a request program requisition in Store A and sent to Store B. 
- Go to store B and click on the response requisition
- See that edibility is blocked

## 💌 Any notes for the reviewer?


## 📃 Documentation

_No user facing changes_

or
